src/index.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * Represents the object returned by the Checkout creation API call.
+ * @see https://api.prod.easypay.pt/docs#tag/Checkout/paths/~1checkout/post
+ */
+interface CheckoutManifest {
+  id: string
+  session: string
+  config: null | {
+    language?: string
+    logoUrl?: string
+    openBtn?: string
+    closeBtn?: string
+    primaryButtons?: string
+    secondaryButtons?: string
+    logoImg?: string
+    stepBadgePrimary?: string
+    stepBadgeSecondary?: string
+    stepTextPrimary?: string
+    stepTextSecondary?: string
+    cartBtn?: string
+    cart?: string
+    cartItem?: string
+    hideDetails?: boolean
+  }
+}
+
+/**
+ * The possible options to configure the Checkout SDK.
+ */
+interface CheckoutOptions {
+  /** The id of the HTML element in which to insert the Checkout form. */
+  id?: string
+  /** The callback to call on Checkout events. */
+  onMessage?: (message: string) => void
+  /** Whether the SDK should use testing APIs. */
+  testing?: boolean
+}
+
+/** The default values for all the options. */
+const defaultOptions: CheckoutOptions = {
+  id: 'easypay-checkout',
+  onMessage: () => { /* do nothing */ },
+  testing: false,
+}
+
+const LOGTAG = '[easypay Checkout SDK]'
+
+/** Validates the necessary parameters for Checkout initialization and gives helpful messages for integrators. */
+function validateParameters(manifest: CheckoutManifest, options: CheckoutOptions) {
+  if (!manifest || typeof manifest.id !== 'string' || manifest.id === '' || typeof manifest.session !== 'string' || manifest.session === '') {
+    console.error(`${LOGTAG} Please provide a valid Checkout Manifest when calling startCheckout.`)
+    return false
+  }
+  if (typeof options.id !== 'string' || options.id === '') {
```

Comment:
I think it would be a good idea to see if exist an element with this id on page and raise an error if not. What do you think?